app/src/main/res/values/strings.xml

```diff
@@ -93,23 +93,23 @@
     <string name="generic_loading_label_1">Passing the butter</string>
     <string name="generic_loading_label_2">Downloading more RAM</string>
     <string name="generic_loading_label_3">Still faster than an iPhone</string>
-    <string name="generic_loading_label_4">What\'s that behind you?</string>
+    <string name="generic_loading_label_4">What's that behind you?</string>
```

Comment:
Quotes need to be escape with `\`. It does not display on the device screen.

Reply:
Okay. Got it. I have seen something like this when passing Windows commands.